### DISCUSSION TITLE
Is there a way to use these files in the original game as a mod?

### DISCUSSION THREAD

**Discussion by lpcysmi** [GalaxyShad/Zuma-Deluxe-HD#15] [General]
Is there a way to use these files in the original game as a mod?

I have the original game on steam but I would love it in better resolution like you have made here

**Comment by GalaxyShad** [comment]
> Is there a way to use these files in the original game as a mod?
> 
> 
> 
> I have the original game on steam but I would love it in better resolution like you have made here
> 
> 
> 
> 

I'm sorry, but adding HD textures to the original Zuma Deluxe game is not possible as it requires changing the game's code and programming. Our fan game is a separate project that utilizes HD textures and other features that are not present in the original game. While it may not be possible to add these features to the original game, you can still enjoy our fan game as a new and improved version of Zuma Deluxe.
